code/game/machinery/doors/airlock.dm

```diff
@@ -696,6 +696,7 @@
 		. = airlock_overlays[iconkey] = mutable_appearance(icon_file, icon_state)
 
 /obj/machinery/door/airlock/proc/check_unres() //unrestricted sides. This overlay indicates which directions the player can access even without an ID
+	cut_overlays()
```

Comment:
Do the cut at the start of update_icons so that it doesnt fuck up overlays implemented in the future that get added before this proc is called